Project3/index.html

```diff
@@ -0,0 +1,118 @@
+<meta charset="UTF-8">
```

Comment:
The beginning of your HTML file got borked somehow. Above line 1, you should have:
```
<!DOCTYPE html>
<html lang="en">

<head>
```